setuptools/tests/config/test_apply_pyprojecttoml.py

```diff
@@ -98,7 +98,9 @@ def test_apply_pyproject_equivalent_to_setupcfg(url, monkeypatch, tmp_path):
   {name = "Tzu-Ping Chung"}
 ]
 maintainers = [
-  {name = "Brett Cannon", email = "brett@python.org"}
+  {name = "Brett Cannon", email = "brett@python.org"},
+  {name = "John X. Ãørçeč", email = "john@utf8.org"},
+  {name = "Γαμα קּ 東", email = "gama@utf8.org"},
```

Comment:
I wonder if it'd handle IDNs as well. I'd add a test case for that too.

Reply:
@abravalheri turned out that the international emails aren't supported in the stdlib, so I thought it'd be reasonable to document that as an xfailing test case: https://github.com/pypa/setuptools/pull/3670.

Reply:
Thank you very much @webknjaz